SPH_SIMULATIONS.md

```diff
@@ -0,0 +1,65 @@
+## Running an SPH Simulation
+
+For **power users** who know how to configure a specific simulator and prepare the respective simulation files, **Inductiva API** provides a simple interface to simulate on a large scale. Here, we demonstrate how to use two SPH simulators in **Inductiva API**.
```

Comment:
While I understand your point of setting  **Inductiva API client**, I think keeping it simple as it was is more clean and still includes everything.

Further, in the new line 5, `otherwise` seems contradictory. I think "Users are, in any case, invited..."